src/flow/flowtests/stylesheet.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+/* eslint-disable no-dupe-keys */
+import * as zacs from '../../index'
+
+const noop = (..._args: any[]): void => {}
+
+const centered = {
+  alignItems: 'center',
+  justifyContent: 'center',
+}
+
+const stylesheet = zacs.stylesheet({
+  root: {
+    _mixin: centered,
+    margin: 0,
+    // $FlowExpectedError[incompatible-call]
```

Comment:
it's not super great, since it won't actually error if error occurs - only emit a warning, which are hidden by default. So you have to run flow with `-max-warnings=0` to make this work

Reply:
Still, it's good we have tests for typings. It aligns nicely with current practices in the community